### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by MaxBorovique**
our stream fixes

### CODE REVIEW

File: src/components/Container/Container.module.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import '../../App.scss';
 
 .container {
-  @include page-spacing;
+  // @include page-spacing;
```
Comment (coldshein) [inline src/components/Container/Container.module.scss:4]: remove comments

File: src/components/Footer/Footer.module.scss [hunk 2]
```diff
@@ -81,12 +80,17 @@
     gap: 16px;
     align-items: center;
 
-    .back_to_top_title {
-        font-size: 12px;
-        color: #7A7D8C;
-    }
+}
+
+.back_to_top_title {
```
Comment (coldshein) [inline src/components/Footer/Footer.module.scss:85]: we already have mixin 'small-text'

File: src/components/Footer/Footer.module.scss [hunk 3]
```diff
@@ -81,12 +80,17 @@
...
+  font-size: 12px;
+  color: #7A7D8C;
```
Comment (coldshein) [inline src/components/Footer/Footer.module.scss:87]: use variable pls

**Review by coldshein — COMMENTED** [review]
Great fixes, please check the comments that I've written below

File: src/components/Container/Container.module.scss (hunk 1 above)
Comment (MaxBorovique) [reply src/components/Container/Container.module.scss:4]: done

**Review by coldshein — APPROVED** [review]

**Pull request merged**